src/main/java/BlackjackController.java

```diff
@@ -3,230 +3,207 @@
 import javafx.scene.control.Label;
 import javafx.scene.control.TextField;
 
+import java.awt.*;
+import java.io.IOException;
 import java.util.List;
 
 public class BlackjackController {
 
     @FXML
-    List<Label> myCardsLabels, dealerCardsLabels;
+    List<Label> playerCards, dealerCards;
     @FXML
-    TextField betAmount;
+    TextField betTextField;
     @FXML
-    Label totalBetLabel, myCardsTotalLabel, dealerCardsTotalLabel,
-            messageLabel, earningsLabel;
-    int totalBet = 0;
-    int earnings = 0;
-    int myCardsTotal = 0;
-    int dealerCardsTotal = 0;
-    int myOnes = 0;
-    int dealerOnes = 0;
-    int dealerFaceDown = 0;
-    boolean gameOver = false;
-    boolean dealerBlackjack = false;
-    boolean meBlackjack = false;
-
-    private final Deck deck;
-
-    public BlackjackController(Deck deck) {
-        this.deck = deck;
+    Label playerCardsTotalLabel, dealerCardsTotalLabel,
+            messageLabel, totalEarningsLabel;
+    double betAmount;
+    double earnings, totalEarnings;
+    int playerTotal;
+    int dealerTotal;
+    int playerAces;
+    int dealerAces;
+    int dealerFaceDown;
```

Comment:
You can put these things in the controller, but things like this belong in a Dealer class, Player class, Hand class...